lib/v8.js

```diff
@@ -397,6 +398,31 @@ function deserialize(buffer) {
   return der.readValue();
 }
 
+/**
+ * @param {{
+ *   filename: string,
+ *   duration: number,
+ * }} [options]
+ */
+function takeGCProfile(options) {
+  validateObject(options, 'options');
+  validateString(options.filename, 'options.filename');
+  validateNumber(options.duration, 'options.duration', 1);
+  return new Promise((resolve, reject) => {
+    const profiler = new binding.GCProfiler();
+    try {
+      profiler.startGCProfile(options.filename);
```

Comment:
Nit: `profiler.startGCProfile` repeats the word profile. Removing "profile" from the function name would be better, I suppose.

Reply:
Fixed, thanks!